gammapy/modeling/models/spatial.py

```diff
@@ -510,6 +510,7 @@ class PointSpatialModel(SpatialModel):
     ----------
     lon_0, lat_0 : `~astropy.coordinates.Angle`
         Center position
+        Default is 0,0 deg
```

Comment:
```suggestion
        Default is "0 deg" , "0 deg"
```

Reply:
minor question: are the quotes needed? As all the rest of the defaults go without them